hint.py

```diff
@@ -7,12 +7,13 @@
 class Hint:
     def __init__(self):
         self.img = None
-        self.cxy = (780, 150)
+        self.cxy = (g.sx(23), g.sy(5))
         self.ctry_idx = 0
         self.hint = ''
         self.ctry = ''
         self.text = 'Click Show button to get hint!'
-        self.textxy = (530,500)
+        self.textxy = (g.sx(17),g.sy(15))
+        self.flagxy = (g.sx(5),g.sy(2))
```

Comment:
Underscores separating the xy from the other parts would help make this more readable.